third_party/rules_pycross/pycross/private/wheel_library.bzl

```diff
@@ -119,6 +129,34 @@ and py_test targets must specify either `legacy_create_init=False` or the global
 This option is required to support some packages which cannot handle the conversion to pkg-util style.
             """,
         ),
+        "patch_args": attr.string_list(
+            default = ["-p0"],
+            doc =
+                "The arguments given to the patch tool. Defaults to -p0, " +
+                "however -p1 will usually be needed for patches generated by " +
+                "git. If multiple -p arguments are specified, the last one will take effect.",
+        ),
+        "patch_dir": attr.string(
+            default = "",
+            doc = "The directory from which to invoke the patch_tool.",
+        ),
+        "patch_tool": attr.string(
+            doc = "The patch(1) utility from the host to use. " +
```

Comment:
We probably should print a warning or add a warning in the docs saying that setting `patch_tool` will mean that builds are not completely hermetic.

I think I am fine with them being that way, but we should do a good job to communicate to the users that.

Reply:
Done